tests/commands_test.go

```diff
@@ -0,0 +1,58 @@
+package tests
```

Comment:
nit: This file should probably be called `list_test.go`?

Reply:
I was split mind here. We have 39 commands and i was thinking do we create 39 separate tests or do one test for all the commands. I'll rename.